nototools/generate_dingbats_html.py

```diff
@@ -512,11 +512,19 @@ def add_index_list_or_defined(info, index, fallback, defines):
   return target_data
 
 
-def generate_text(outfile, title, fonts, targets, data_dir):
-  emoji_only = (
-      unicode_data.get_emoji() - unicode_data.get_unicode_emoji_variants(
-          'proposed_extra'))
-
+def _flagged_name(cp, flag_sets):
+  """Prepend any flags to cp's unicode name, and return."""
+  name = unicode_data.name(cp)
+  flags = []
+  for k, v in sorted(flag_sets.iteritems()):
+    if (cp in v[0]) == v[1]:
+      flags.append(k)
```

Comment:
How about repeating this comment taken from  |flag_sets| here? 
"# True means set flag if cp in set, False means set if not in set"